### ISSUE TITLE
Examples do not work

### ISSUE THREAD

**Issue by future-xy** [kserve/modelmesh-serving#156]
**Many examples do not work**

Hi,

Thanks for providing modelmesh-serving! I'm trying to use examples provided in ./config/example-predictors/ but I found only *example-sklearn-mnist-svm* can be accessed. Other examples, such as *example-onnx-mnist, example-tensorflow-mnist, example-keras-mnist* can't be used. Specifically, clients (both grpc and REST) return "predict() method not implemented" error.

**To Reproduce**

1. Follow the example in quickstart.md but change the model from *example-mnist-predictor* to others, such as *example-onnx-mnist*
2. Run:
```
MODEL_NAME=example-tensorflow-mnist
curl -X POST -k http://localhost:8008/v2/models/${MODEL_NAME}/infer -d '{"inputs": [{ "name": "predict", "shape": [1, 64], "datatype": "FP32", "data": [0.0, 0.0, 1.0, 11.0, 14.0, 15.0, 3.0, 0.0, 0.0, 1.0, 13.0, 16.0, 12.0, 16.0, 8.0, 0.0, 0.0, 8.0, 16.0, 4.0, 6.0, 16.0, 5.0, 0.0, 0.0, 5.0, 15.0, 11.0, 13.0, 14.0, 0.0, 0.0, 0.0, 0.0, 2.0, 12.0, 16.0, 13.0, 0.0, 0.0, 0.0, 0.0, 0.0, 13.0, 16.0, 16.0, 6.0, 0.0, 0.0, 0.0, 0.0, 16.0, 16.0, 16.0, 7.0, 0.0, 0.0, 0.0, 0.0, 11.0, 13.0, 12.0, 1.0, 0.0]}]}'
```
3. Result:
```
{"code":2,"message":"inference.GRPCInferenceService/ModelInfer: UNKNOWN: Unexpected <class 'NotImplementedError'>: predict() method not implemented"}
```

**Environment:**

- OS: Ubuntu 20.04

**Comment by njhill** [comment]
@future-xy it looks like the model was loaded in the wrong runtime (MLServer). By deafult, tensorflow and onnx models will run in Triton. This could be related to an older bug that should be fixed in the most recent 0.8 release. Could you confirm which version you're using?

Could you also share details of what pods are running, and the state/status of the predictor in question (`kubectl describe predictor example-tensorflow-mnist`)

**Comment by future-xy** [comment]
Thanks for your reply! The problem has disappeared after I reinstalled the 0.8 release.
